### DISCUSSION TITLE
Il tema del libro è stato deciso!

### DISCUSSION THREAD

**Discussion by Cadienvan** [Il-Libro-Open-Source/book#4] [Annunci]
Un piccolo passo è stato fatto, ed è giusto condividerlo con tutti voi.

Il nostro libro ha finalmente un tema: **Il Manuale del buon dev**.

Come deciso da ben 17 votanti nell'apposito survey #2 , la prima mattonella è stata posta.

Ringrazio tutti voi per aver partecipato a questo primo passo e spero rimarrete accanto a me per tutti gli altri che faremo assieme.

**Comment by guizzo** [comment]
Archivio la discussione in quanto ormai datata
